test/spec/modules/ucfunnelBidAdapter_spec.js

```diff
@@ -89,34 +89,34 @@ const validNativeBidRes = {
   width: 1
 };
 
-describe('ucfunnel Adapter', function () {
-  describe('request', function () {
-    it('should validate bid request', function () {
+describe('ucfunnel Adapter', () => {
+  describe('request', () => {
```

Comment:
Please do not use Arrow functions as recommended by mocha. 
See here https://github.com/prebid/Prebid.js/issues/2987